### DISCUSSION TITLE
How can I reach photoprism remotely?

### DISCUSSION THREAD

**Discussion by marcijel1** [photoprism/photoprism#3057] [Q&A] [answered]
Hello everyone!

I need help about this.

I found this recommendation on Photoprism site
[https://docs.photoprism.app/getting-started/proxies/traefik/](https://docs.photoprism.app/getting-started/proxies/traefik/)
But I can't really understand what should I do with it.

I installed Photoprism perfectly, it's working, but only reachable from LAN.
My setup is the following:
Ubuntu 22.10
I reach the server from a DDNS address.

How can I reach Photoprism remotely?

I need something step-by-step instructions, because this is a totally new topic for me.

Thank you for your help.

**Comment by marcijel1** [comment, marked as answer]
this video helped me
[https://www.youtube.com/watch?v=wLrmmh1eI94](https://www.youtube.com/watch?v=wLrmmh1eI94)

Also my problems was:
-snap version of docker is limited. For photoprism it's working normally I think, but other apps, like traefik, just can't write to lot of places. So snap version of docker is a shit, not recommended.
-traefik can't create the certificate file (don't know why, maybe just for me). I had to made it manually, simply just a blank cert.json file with permission 600. then it started working.

Now the redirection working fine, and I have the httpS 😎

**Reply by lastzero** [reply]
Glad you have it working! 👍
